lib/puppet/provider/service/rcng.rb

```diff
@@ -0,0 +1,50 @@
+Puppet::Type.type(:service).provide :rcng, :parent => :bsd do
+  desc <<-EOT
+    RCng service management with rc.d
+  EOT
+
+  defaultfor :operatingsystem => [:netbsd, :cargos]
```

Comment:
@jasperla could you update this PR to `confine` to `netbsd` and `cargos`, otherwise, this provider may be suitable, and chosen on other platforms.